library/spdm_crypt_lib/libspdm_crypt_cert.c

```diff
@@ -739,12 +739,14 @@ static bool libspdm_verify_cert_subject_public_key_info(const uint8_t *cert, siz
  *
  * @param[in]  cert                  Pointer to the DER-encoded certificate data.
  * @param[in]  cert_size             The size of certificate data in bytes.
+ * @param[in]  must_have_basic_constraints  This value indicates whether basic_constraints must be present in the Cert
```

Comment:
`must_have_basic_constraints` -> `need_basic_constraints`

Reply:
Thanks. I have changed the variable name.